docs/operate-a-stake-pool/guild-ops-suite.md

```diff
@@ -64,8 +74,8 @@ Issue/PRs are welcome on the [github repository][guild-github].
 
 ### Support Requests
 
-We do have [telegram channel for support requests][guild-tg] , but note that we intend to have support channel only for baseline skillset highlighted on [homepage][guild-website].  
+We do have [telegram channel for support requests][koios-general-tg] , but note that we intend to have support channel only for baseline skillset highlighted on [homepage][guild-website].  
```

Comment:
Would help I think to make an explanatory note that the Telegram link points to _Koios Discussions_; otherwise it looks like it could be an error.